.github/workflows/check.yml

```diff
@@ -17,40 +17,40 @@ jobs:
       fail-fast: false
       matrix:
         py:
+          - "3.11.0-beta.5"
```

Comment:
```suggestion
          - "3.11-dev"
```

Reply:
No thanks.

Reply:
Why not [rc1](https://www.python.org/download/pre-releases)?

Reply:
Until aiofiles makes a new release doesn't matter. You were suggesting dev though not rc1.

Reply:
Understood.  `dev == rc1` until there is an `rc2` ;-)